app/views/time_regs/index.html.erb

```diff
@@ -1,2 +1,5 @@
+<%- if @active_time_reg %>
+  <%= render BannerComponent.new(title: "Notice", message: t("time_regs.active_timer_description"), variant: :warning, dismissible: false, action: { url: root_path(date: @active_time_reg.date_worked), method: :get }) %>
+<% end %>
```

Comment:
Can you make it so the banner doesn't push down the content below?

Reply:
You mean it overlaps content instead of being place on top it?

Reply:
Because if that's the case, I purposely chose the banner to push down content and not overlap any information.
Unless I'm missing some details in your feedback

<img width="1726" alt="Screenshot 2024-05-07 at 06 37 02" src="https://github.com/rubynor/reap/assets/42216593/076b9d7d-0782-4a22-832f-64913a50a53f">

Reply:
I didn't know this was intentional. Never mind this comment then :sweat_smile: